backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -364,7 +374,19 @@ public String verifyOrganizationNoPermissions(String externalId) {
     }
     org.setIdentityVerified(true);
     organizationRepository.save(org);
-    return oktaRepository.activateOrganizationWithSingleUser(org);
+    if (featureFlagsConfig.isOktaMigrationEnabled()) {
+      Optional<ApiUser> orgAdmin =
+          dbAuthorizationService.getOrgAdminUsers(org).stream().findFirst();
+
+      if (orgAdmin.isPresent()) {
+        String orgAdminEmail = orgAdmin.get().getLoginEmail();
+        return oktaRepository.activateUser(orgAdminEmail);
+      } else {
+        throw new IllegalStateException("Organization does not have any org admins.");
```

Comment:
⚠️ This `else` block changes the behavior of this method

Previously, even if the org had [no admin users this method would succeed](https://github.com/CDCgov/prime-simplereport/pull/8105/files#diff-92fa7f1f6942c235dcf316810541a74e42e432f1798b75f749fd4a273891c4f4R335)

However, with this implementation if an org does not have an admin it will [throw an exception](https://github.com/CDCgov/prime-simplereport/pull/8105/files#diff-92fa7f1f6942c235dcf316810541a74e42e432f1798b75f749fd4a273891c4f4R348).

Let me know if you think this should continue to have the same behavior as before. 🤔

Reply:
I think this makes sense. Btw do we have any additional context on the comment on this method? Is this an old reference to some account request workflow that hasn't been implemented?

```
/**
   * This method is for verifying an organization after the Experian identity verification process.
   * It should not be used for any other purpose and once we move to the updated account request
   * workflow this should be removed.
   */
  @Transactional(readOnly = false)
  public String verifyOrganizationNoPermissions(String externalId) {
```

Reply:
That's a good question... Let me check that commit history in see if I can uncover anything... 🕵️

Reply:
This is the PR that introduced this change: https://github.com/CDCgov/prime-simplereport/pull/2096
I did a search in the repo and couldn't find any mention of a new or updated account request flow 🤔 so I'm not entirely sure what that comment is referencing....